app/src/main/java/com/sdp/movemeet/UploadActivityActivity.java

```diff
@@ -344,7 +363,6 @@ private Activity validateActivity() {
         validDate = true;
 
         String organizerId = FirebaseAuth.getInstance().getCurrentUser().getUid();
-
         Activity activity = new Activity(
```

Comment:
Do you think it will be the right place to also create the "ActivityChatID" for identifying the chat corresponding to this specific sport activity? I'm interested in this since that's certainly what I'm going to work on next week ^^

Reply:
Yeah sure !

Reply:
Yes, it will be here! I will create an additional field for the "ActivityChatID" then! 👍